### DISCUSSION TITLE
[11.0.0 rc1.1] Texts are displaced up on Linux, working fine on Win11

### DISCUSSION THREAD

**Discussion by alexandrehtrb** [AvaloniaUI/Avalonia#11661] [Q&A] [answered]
**Describe the bug**
Hello,

I am migrating my program to Avalonia 11.0.0 rc1.1 and the texts are displaced with a "margin bottom" on Linux, but they are appearing fine on Windows.

**To Reproduce**

Code if you want to clone:
https://github.com/alexandrehtrb/Pororoca/tree/develop

**Expected behavior**
The appearance and layout should be the same on both platforms.

**Desktop (please complete the following information):**

- OSes: Arch Linux x64 with GNOME, Windows 11 x64

**Screenshots**

On Linux:

![Pororoca_linux_gnome_font_placement](https://github.com/AvaloniaUI/Avalonia/assets/27026741/9978ecb1-20f0-4f9f-9b96-d3c0b62af908)

On Windows:

![Pororoca_win11_font_placement](https://github.com/AvaloniaUI/Avalonia/assets/27026741/f8729945-7934-47d5-a3ed-4fe727e96762)

**Comment by Gillibald** [comment]
You are probably not shipping a default font with your app

**Comment by maxkatz6** [comment, marked as answer]
Note, if you want to include Inter font (which was included by default in 0.10), then you need to add `Avalonia.Fonts.Inter` package and add `.WithInterFont()` to your app builder.

**Reply by cok666n** [reply]
Both your comment and the porting guide mention UseInterFont(), but I could only find the .WithInterFont() method, is that an error in the guide or Im the one that's mixing things up ?

```
  public static AppBuilder BuildAvaloniaApp()
            => AppBuilder.Configure<App>()
                .UsePlatformDetect()
                .WithInterFont()
                .LogToTrace()
                .UseReactiveUI()
                ;
```

**Reply by timunie** [reply]
probably a typo. can you send a PR to the wiki?

**Reply by cok666n** [reply]
Seems someone already took care of it, all good!

**Comment by robloo** [comment]
This issue still looks like a bug to me. I dont ever recall WPF changing baseline measurement quite like this based on font. Additionally, I'm curious if TextBox has the same issue. If textbox centers correctly definitely seems like a bug.

See #10658
